frontend/package.json

```diff
@@ -66,6 +66,7 @@
         "karma-jasmine-html-reporter": "~1.7.0",
         "prettier": "^2.7.1",
         "prettier-eslint": "^15.0.1",
-        "typescript": "^4.6.4"
+        "typescript": "^4.6.4",
+        "webpack": "^5.74.0"
```

Comment:
Why did you add this?

Reply:
Added for fix - warning "@angular/fire > file-loader@6.2.0" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0". Without this package we have warning